src/helpers/SdDaemon.hpp

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+int sd_booted(void);
```

Comment:
if we are rewriting this, some style things to consider.

Firstly, this could be wrapped in a namespace e.g. `namespace Systemd`

Secondly, no_snake, we use camelCase